framework/projects/Closure/compile-errors/test-134-154.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/test/com/google/javascript/jscomp/DataFlowAnalysisTest.java b/test/com/google/javascript/jscomp/DataFlowAnalysisTest.java
+index e07b13513..0dd914e4b 100644
+--- a/test/com/google/javascript/jscomp/DataFlowAnalysisTest.java
++++ b/test/com/google/javascript/jscomp/DataFlowAnalysisTest.java
+@@ -659,7 +659,7 @@ public class DataFlowAnalysisTest extends TestCase {
+         assertTrue(outEdges.size() < 2);
+         ConstPropLatticeElement aResult = flowThroughArithmeticInstruction(
+             (ArithmeticInstruction) node, input);
+-        for (DiGraphEdge<Instruction, Branch> _ : outEdges) {
++        for (int i = 0; i < outEdges.size(); i++) {
```

Comment:
To minimize the differences, how about renaming the identifier from `_` to `e` or `edge`?

The identifier isn't used, so the name does not really matter.

Reply:
I just copied the upstream fix from apache.  But I happy to make your change instead.  You pick.

Reply:
Even better :)